k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4EDM4hep2LcioConv.ipp

```diff
@@ -833,6 +825,27 @@ namespace EDM4hep2LCIOConv {
       }
 
     } // SimTrackerHits
+
+    // Resolve relations for clusters
+    for (auto& [lcio_cluster, edm_cluster] : update_pairs.clusters) {
+      for (const auto& edm_linked_cluster : edm_cluster.getClusters()) {
+        if (edm_linked_cluster.isAvailable()) {
+          if (
+            const auto lcio_cluster_linked =
+              k4EDM4hep2LcioConv::detail::mapLookupFrom(edm_linked_cluster, lookup_pairs.clusters)) {
+            lcio_cluster->addCluster(lcio_cluster_linked.value());
+          }
+        }
+      }
+
+      for (const auto& edm_calohit : edm_cluster.getHits()) {
+        if (edm_calohit.isAvailable()) {
+          if (const auto lcio_calohit = k4EDM4hep2LcioConv::detail::mapLookupFrom(edm_calohit, update_pairs.caloHits)) {
+            lcio_cluster->addHit(lcio_calohit.value(), 0);
```

Comment:
EDM4hep doesn't have a weight for the calo hits(?). What is the proper value to set here?

Reply:
Tracking this in #key4hep/EDM4hep#236